rust/otap-dataflow/crates/otap/src/encoder.rs

```diff
@@ -3728,6 +3728,42 @@ mod test {
         _test_attributes_all_field_types_generic(RawLogsData::new(&logs_data_bytes));
     }
 
+    #[test]
```

Comment:
This test was added due to a bug in our test where we were trying to see `batch_length` for logs. It seems to be a timing issue, and this can be removed if thought superflous.